src/app.js

```diff
@@ -0,0 +1,40 @@
+// isYes function
```

Comment:
Using comments like this helps a lot for outlining out your code while you're working on it! I totally encourage using this to organize your thought process as you go. However, you need to delete any comments out of your code before you submit your lab for grading, and starting next week, we'll start taking points off for this. If you're concerned about losing any comments, I just recommend committing before you delete them out so that you can refer back to that commit to see your comments.